pkg/gardenadm/botanist/controlplane.go

```diff
@@ -130,3 +134,37 @@ func (b *AutonomousBotanist) populateStaticAdminTokenToAccessTokenSecret(ctx con
 
 	return b.SeedClientSet.Client().Update(ctx, secret)
 }
+
+// CreateClientSet creates a client set for the control plane.
+func (b *AutonomousBotanist) CreateClientSet(ctx context.Context) (kubernetes.Interface, error) {
+	clientSet, err := kubernetes.NewClientFromFile("", PathKubeconfig,
+		kubernetes.WithClientOptions(client.Options{Scheme: kubernetes.SeedScheme}),
+		kubernetes.WithClientConnectionOptions(componentbaseconfigv1alpha1.ClientConnectionConfiguration{QPS: 100, Burst: 130}),
+	)
+	if err != nil {
+		b.Logger.Info("Waiting for kube-apiserver to start", "error", err.Error())
```

Comment:
```suggestion
		b.Logger.Info("Creating client set failed", "error", err.Error())
```

Reply:
This is the technical error reason, but I think it's more user-friendly to keep what we have here. Once KAPI started, the client set can be created. While this is just a log/eye-candy, the error returned a line later contains the technical error reason. WDYT?